demonstrations/tutorial_game_of_surface_codes.py

```diff
@@ -0,0 +1,394 @@
+    If all :math:`X` measurements on qubits 1-4 in step 23 yield a :math:`+1` result, a magic state is successfully prepared on qubit 5. The probability for failure is roughly :math:`(1-p)^n`.
+    **
+
+The 15-to-1 distillation protocol produces a magic state in 11🕒 on 11 tiles.
+
+Quantum computer designs
+------------------------
+
+The 15-to-1 distillation protocol is the simplest of a variety of protocols with each their different characteristics.
+The best choice for distillation protocol heavily depends on the error probabilities of the quantum computer in use,
+as well as the overall tolerance for errors we allow to still occur.
+For example, assume we tolerate T infidelity of :math:`10^{-10}` and have :math:`p=10^{-4}`, then
+the 15-to-1 protocol would suffice as it yields an infidelity of :math:`35p^3 = 3.5 \times 10^{-11} < 10^{-10}`.
+
+Another consideration is to combine data and distillation blocks that match in their maximum time requirements.
+Since the 15-to-1 distillation above takes 11🕒 to procude a magic state, there is no point in using the fast or intermediate data blocks, and we can just resort to the compact one.
+
+A minimal setup can be seen below. It consists of 100 logical qubits on 153 tiles in a compact block, as well as a 15-to-1 distillation block using another 11 tiles.
+
+.. figure:: ../_static/demonstration_assets/game_of_surface_codes/minimal-setup.png
+    :align: center
+    :width: 50%
+    :target: javascript:void(0)
+
+    Minimal setup with 100 logical qubits on 153 tiles and 11 extra tiles for a compact distillation block.
+    **
+
+For a code distance of :math:`d=13` we would require :math:`164 \cdot 2 \cdot d^2 \approx 55k` physical qubits.
+An example computation with :math:`10^8` T gates at a code cycle of :math:`1\mu s` would finish in :math:`d \cdot 11🕒 \cdot 10^8 \approx 4h`.
+
+In this setup, a magic state is produced every 11🕒 and takes at most 9🕒 for consumption.
+The bottleneck is in the magic state distillation, and overall this setup takes 11🕒 per non-Clifford gate.
+The most straight-forward way to speed this up is by adding magic state distillation blocks. Adding just one other distillation block halves the T-gate production time to 5.5🕒.
+Now it makes sense to use the intermediate data block design, which takes at most 5🕒 for T-gate consumption:
+
+.. figure:: ../_static/demonstration_assets/game_of_surface_codes/intermediate_setup.png
+    :align: center
+    :width: 70%
+    :target: javascript:void(0)
+
+    Intermediate setup consisting of the intermediate data block and two 15-to-1 distillation blocks on each end.
+    **
+
+In this case we require 222 tiles, so :math:`222 \cdot 2 \cdot d^2 \approx 75k` physical qubits, and the same computation mentioned before would finish in half the time after about :math:`2h`.
+
+
+Conclusion
+----------
+
+We've been introduced to a high-level description that allows us to reason about space-time trade-offs in FTQC architecture designs.
```

Comment:
... high-level description of what? 
```suggestion
We've been introduced to a high-level description that allows us to reason about space-time trade-offs in FTQC architecture designs.
```

Reply:
.. of quantum computing I guess ✅ :)